### PR TITLE
Update Python version to 3.6.13 to avoid cache poisoning vulnerability

### PR DESCRIPTION
**Pull request by jacobtoppm**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ isort:
 # Check settings, migrations and run tests.
 test:
   # Make sure this Python version matches the version in your Dockerfile.
-  image: python:3.6.6
```
Comment (zerolab) [inline .gitlab-ci.yml:74]: why not leave this at `image: python:3.6` (and ditto below)?
this means you'd only need to rebuild the container to get the newer version

Comment (jacobtoppm) [reply .gitlab-ci.yml:74]: Sounds fine to me, was just following what we'd done previously in terms of specifying a point release. I'll tweak it.

Comment (jacobtoppm) [reply .gitlab-ci.yml:74]: Changed in latest commit.

**Review by zerolab — APPROVED** [review]
LGTM

**Pull request merged**